pkg/build/controller/build/build_controller.go

```diff
@@ -1087,6 +1092,21 @@ func (bc *BuildController) resolveImageReferences(build *buildv1.Build, update *
 	return nil
 }
 
+func retryOnOwnerRef(build *buildv1.Build, pod *corev1.Pod) error {
+	if len(pod.OwnerReferences) > 0 {
```

Comment:
A general rule is to return early, the code will read better if this looks like:
```
if len(pod.OwnerReferences) == 0 {
    return nil
}
// ... rest of the code
```